rtublen/Studentportalen
# Features/us1227 add profile page

Adds profile page for Robert Tublén.

## How to verify change

1. Start application.
2. Open /profiles/roberttublen
3. Verify that you see profile page for Robert Tublén.
